vmm/src/config.rs

```diff
@@ -2188,6 +2193,14 @@ impl VmConfig {
                 return Err(ValidationError::CpuTopologyZeroPart);
             }
 
+            // The setting of dies doesen't apply on AArch64.
+            // Only '1' value is accepted, so its impact on the vcpu topology
+            // setting can be ignored.
+            #[cfg(target_arch = "aarch64")]
+            if t.dies_per_package != 1 {
```

Comment:
This is the easiest way I figured out to get rid of `dies` setting on AArch64.

Alternative solution could be removing this field on AArch64 and renaming some other fields. But I think it's better to keep the parameters format aligned between architectures as much as possible.

What do you think? @sboeuf @rbradford

Reply:
lgtm.